ecl/regress/signedtests/regress.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+###############################################################################
+#  HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+##############################################################################
+# Instructions:
+#   Same as options as regress.sh in parent directory
+#   (calls regress.sh in parent directory)
+../regress.sh $* -x "--allowsigned datafile"
```

Comment:
I don't think this is really testing the code, since with this option I don't think it would fail if it wasn't tested.  I tested with

`-x '--deny=all' -x '--allow=userECL' -x '--allow=extern' -x '--allow=cpp' -x '--allow=pipe' -x --allowsigned=datafile`